packages/core/contracts/NFTContract.test.ts

```diff
@@ -115,4 +115,45 @@ describe('Transfer NFTs', () => {
     // The 3 NFTs should now be in the new account
     expect([onChainNFT1.id, onChainNFT2.id, onChainNFT3.id]).toEqual([nft1.id, nft2.id, nft3.id]);
   });
+
```

Comment:
Can you change the name of this test suite to something like `describe('Common NFT tests'...`? It's at the top of the file.